news/_posts/2020-12-09-pr-testing-on-prestashop.md

```diff
@@ -0,0 +1,171 @@
+Of course, you’re welcome to fix issues submitted by another contributor (as long as the issue is linked to the PR) : This is the magic of open source !
+- The “How to test” field must be filled. It must contain the steps to check the bug is indeed fixed. If possible, it should also contain some information about the possible side-effects of the PR, in order to help check if the PR does not introduce regressions. Without a clear “how to test”, it’s impossible for us to know what to check on the PR.
+The most important thing about this part is that it must be a functional “How to test” :  PR are tested by functional manual testers, not technical ones. If your PR needs to make some technical changes in order to be tested (use a certain hook in the code for example): either make a test module allowing us to test it, or your PR will be tested by the maintainers team.
+
+![alt text](https://github.com/PrestaShop/prestashop.github.io/assets/images/2020/12/clear_howtotest.png "Clear How to Test in a PR")
+
+Other important things :
+- The PR needs to have been reviewed and approved by at least two maintainers.
+- All automatic checks must have passed
+- The PR is made on the good branch
+
+If there is some information missing, the author, the maintainers or the product team is pinged and the tester waits for this information to test the PR.
+
+### Reproduce the bug
+
+The first step of PR testing is reproducing the bug (on the branch, without the PR) : This allows the tester to make sure that the “How to test” is clear and also helps to see what are the possible regressions introduced by the PR. 
+This is why a complete issue is necessary : the more information it contains, the easier it is to reproduce it, and to test it.
+
+### Check if the Pull Request fixes the bug in the issue
+
+This is the most logical part of PR Testing : after reproducing the bug on the branch, the tester installs the PR and checks if the bug is indeed fixed, following the “How to test”.
+
+If the feature is a bit complex, or if the “How to test” is not complete enough, the tester will also test some edge cases in order to make sure that the PR fixes the issue completely.
+
+### Regression testing
+
+This is the most important part of our work : Usually, the nominal test is OK, and has already been tested by the author.
+But the PR might have introduced some regressions somewhere else in PrestaShop. This is where our functional knowledge of the solution is really important :
+
+With the PR, the issue, and the information we can have by looking at the code (for the testers who have a technical background and know how to “read” the code), we must identify the possible side-effects of the PR, and test them all, or at least, the most important of them (since we cannot pass hours and hours on each PR).
```

Comment:
```suggestion
With the PR, the issue, and the information we can have by looking at the code (for the analysts who have a technical background and know how to “read” the code), we must identify the possible side-effects of the PR, and test them all, or at least, the most important of them (since we cannot pass hours and hours on each PR).
```